Sodium/Sign.swift

```diff
@@ -38,7 +38,7 @@ public class Sign {
         return KeyPair(publicKey: PublicKey(data: pk!), secretKey: SecretKey(data: sk!))
     }
     
-    public func keyPair(seed seed: NSData) -> KeyPair? {
+    public func keyPair(#seed: NSData) -> KeyPair? {
```

Comment:
`seed seed` works fine under Xcode 7, so I'd rather keep that since it also works on Xcode 6 minus a warning.